assets/js/util.js

```diff
@@ -190,14 +191,14 @@ function callApy(options /*: any */, endpoint /*: any */, useAjax /*: any */) {
         return $.ajax(requestOptions);
     }
 
-    return $.jsonp(requestOptions);
+    return $.ajax.jsonp(requestOptions);
```

Comment:
I don't think this is correct. `$.jsonp` is a library separate from jQuery.

Reply:
Well, `$.jsonp` didn't resolve, while `$.ajax.jsonp` did.
Shall we stub this?

Reply:
Probably the best idea, yes.